### PR TITLE
Doc updated.

### PR DESCRIPTION
**Pull request by abdulniyaspm**

### CODE REVIEW

**Comment by abdulniyaspm** [comment]
I am not sure if this is the correct wording. For me this title was bit confusing.

File: docs/README.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ $ unimport [sources [sources ...]]
 _If you do not get any output, congratulations means there is no unused import in your
 project._
 
-#### Let's with example with simple a Python code
+#### Let's start with a simple python code
```
Comment (hakancelik96) [inline docs/README.md:46]:
```suggestion
#### Let's start with a simple Python code
```

**Comment by hakancelik96** [comment]
Also, You can remove `_If you do not get any output, congratulations means there is no unused import in your
project._` the line from [README](https://github.com/hakancelik96/unimport/blob/master/docs/README.md), cause Unimport is already showing this message, so this line is unnecessary.

**Review by hakancelik96 — COMMENTED** [review]
I think this sentence is correct but it will be better if you just fix Python to capitalize.

**Comment by abdulniyaspm** [comment]
@hakancelik96  Fixed. Also removed the line `_If you do not get any output, congratulations means there is no unused import in your project._`

**Review by hakancelik96 — APPROVED** [review]

**Comment by hakancelik96** [comment]
Thank you for your contribution.

If you interest in this project, I will open a few issues as soon as possible, I recommend that you look at them when I open them.

**Comment by hakancelik96** [comment]
@abdulniyaspm My mistake I forgot a topic, you should have written your name in the [AUTHORS.md](https://github.com/hakancelik96/unimport/blob/master/docs/AUTHORS.md) file.

Please read [CONTRIBUTING](https://github.com/hakancelik96/unimport/blob/master/docs/CONTRIBUTING.md) next time and do not forget to write your name, if you want you can send a new PR for it.

**Comment by hakancelik96** [comment]
Here I opened a new issue to improve documentation https://github.com/hakancelik96/unimport/issues/88

**Pull request merged**